_rules/SC1-1-1-valid-duplicate-accessible-name-for-image.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Valid use of duplicate accessible names for images
+description: |
+	This rule checks that there is a valid reason for having a set of images with identical accessible names.
+
+success_criterion:
+- 1.1.1 # Non-text Content
+
+test_aspects
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+### Applicability
+
+The rule applies to any set of HTML or SVG element with the semantic role of `img`, in a document, exposed to assistive technologies, that have an identical [accessible name]().
+
+### Expectation
+
+Each test target has an identical purpose.
+
+**Note**: Leading and trailing whitespace and difference in case sensitivity should be ignored.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+There is a known combination of a popular browser and assistive technology that does not by default support `title` as an accessible name.
```

Comment:
This is repeated in at least a few rules. Can we at some point make this known combination of browsers and assistive technologies _known_?

Reply:
I would be all for that but I was under the impression that we can't or didn't want to call out vendors. @annethyme do you know what the stance is on that?

Reply:
Correct. The problem is that it only takes one or two tests to figure out something doesn't have wide support. But if you want to report that, it's not fair to the two or three that you tested to only list them as not supporting it, as opposed to all others that probably don't either, but we didn't test that.

We don't have the resources to do comprehensive testing of AT in Auto-WCAG. There are a few promising projects happening that, if they combined resources might be able to do it. I'd love to link to something like that in the future.

Reply:
Makes sense 👍 It'd be fantastic to have something similar to https://github.com/mdn/browser-compat-data for information about AT compatibility.